source/lifecycle/mattermost-v10-changelog.md

```diff
@@ -0,0 +1,14 @@
+# Mattermost v10 changelog
```

Comment:
@amyblais - You're welcome to start populating upcoming v10 release details, such as breaking changes or other advanced notifications, on this new v10 changelog page.